deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -169,6 +169,7 @@ services:
     command:
       [
         "-config=/root/config/prometheus.yml",
+        "-gateway-url=https://api.staging.signoz.cloud"
```

Comment:
Remove it from the general docker-compose (our OSS users use the docker deployment) and pass it wherever you need to enable the gateway setup.